project/static/css/styles.css

```diff
@@ -555,4 +553,23 @@ img {
 .edit, .delete, .restore {
     width: fit-content;
     margin-left: 24px;
+}
+
+
+
+/********** Search bar **********/
+.search-bar {
+    width: 20%;
+}
+.search-bar form {
+    flex-direction: unset;
```

Comment:
Why you set unset value?

Reply:
Because it takes the form's value for display and then search bar is above of search button. Maybe there is a better way to avoid this behaviour 🤔

Reply:
I didn't know. I never use uset beforen, but it sounds good. :)